### DISCUSSION TITLE
No word level transcription

### DISCUSSION THREAD

**Discussion by burak2102** [chidiwilliams/buzz#426] [Q&A]
Hello, I installed this wonderfull app via Mac Appstore and I can’t see word level tick anywhere.

**Comment by chidiwilliams** [comment]
@burak2102 Yes, this is not currently available, but I'll add it in the next release.

**Comment by burak2102** [comment]
thank you 🙏🏻
30 Nis 2023 Paz, saat 04:38 tarihinde Chidi Williams <
***@***.***> şunu yazdı:

**Reply by chidiwilliams** [reply]
Could you try updating now?

**Comment by burak2102** [comment]
Amazing 🙏🏻 The feature has arrived. I want to ask, which version of Buzz is more reliable? The Mac Appstore one or the one in the dmg file?iPad’imden gönderildiChidi Williams ***@***.***> şunları yazdı (30 Nis 2023 14:47):﻿
Could you try updating now?

**Reply by chidiwilliams** [reply]
The App Store version is a lot better. The transcriptions are a bit faster, the app itself is cleaner and easier to use because it's native, the transcript editor is better, and updating from the App Store is a lot easier than manually downloading the DMG file every time.

**Reply by chidiwilliams** [reply]
@burak2102, could you pls help leave a rating on the App Store if you've really liked Buzz?

**Comment by burak2102** [comment]
Ok sure I will 👍🏻

**Reply by chidiwilliams** [reply]
Wanted to send you a reminder. Not sure if you've had a chance to yet: https://apps.apple.com/us/app/buzz-captions/id6446018936?mt=12&itsct=apps_box_badge&itscg=30200

**Comment by burak2102** [comment]
Hello I forgot. and just did it. Please check it out 🙏🏻

**Reply by chidiwilliams** [reply]
Thank you!
